tests/test_runlib.py

```diff
@@ -571,7 +570,7 @@ def test_exclude_patterns(self):
                 ],
             ),
             (
-                ["*esc\!"],  # pylint: disable=W1401
+                ["*esc\\!"],
```

Comment:
The anomalous backslash may be intentional. 🤷 

-> https://github.com/in-toto/in-toto/pull/487#discussion_r809668288

Reply:
@rszamszur, @adityasaky, do you remember the your motivations from 3 years ago? 😅

Reply:
The intent/motivation is irrelevant, this is not a valid escape sequence in Python. In the absence of a valid escape sequence, Python interprets `\` as `\\`, see [2.4.1.1. Escape sequences](https://docs.python.org/3/reference/lexical_analysis.html#escape-sequences):
> Unlike Standard C, all unrecognized escape sequences are left in the string unchanged, i.e., _the backslash is left in the result_. (This behavior is useful when debugging: if an escape sequence is mistyped, the resulting output is more easily recognized as broken.)

The alternative is raw strings:
```suggestion
                [r"*esc\!"],
```

Reply:
Okay, thanks for checking. Let's leave it as you proposed then. The change doesn't seem to break the tests.